### DISCUSSION TITLE
Hii

### DISCUSSION THREAD

**Discussion by Shiv-45** [community/community#34853] [Other Feature Feedback, Questions, & Ideas]
I'm new on GitHub could help me how to contribute project?

**Comment by DhananjayPorwal** [comment]
If you wanna to know how you can create pull request in any public repository then you can check this [README.md](https://github.com/DhananjayPorwal/hacktoberfest2022/blob/main/README.md). 
> PS: If you are satisfied with this answer, select this comment as an answer.

**Reply by Shiv-45** [reply]
Hii
